src/components/ProductCard.vue

```diff
@@ -46,7 +46,7 @@
 
     <div
       v-for="(theInfo, index) in consumerInfo"
```

Comment:
`error: 'index' is defined but never used`
So it needs to be:
`v-for="theInfo in consumerInfo"`
Is there a proper way to suggest or edit that change and then approve the pull request?